docs/nodle-wallets/nodle-cash/faq.mdx

```diff
@@ -77,20 +78,59 @@ This is similar to your email address when people want to send you an email.
 
 You can find your public key in the Nodle Cash app by clicking the "Receive" button below your balance.
 
+### What is a legacy Nodle wallet?
+
+The Nodle Cash app has two wallet systems: legacy and not legacy. Legacy Nodle wallet might be a bit more complex
+to import to other wallets supporting NODL, or it might not be compatible with other wallets at all. Other than that, it
+is a valid wallet, and you can use it as before.
+
+If you have just joined the Nodle network, you have a non-legacy wallet. You can check it if you navigate to the screen 
+with your public key by clicking the "Receive" button. The app will indicate whether you are using a legacy wallet or not.
+If it does not say anything, you are using a non-legacy wallet.
+
+If you use a legacy wallet, we recommend you switch to a new one.
+
+### How can I switch to a new wallet from a legacy one?
+
+The app can automatically upgrade your wallet during the wallet import if you have 0 balance.
+In this case, you will get a new public key, which you need to share with your peers to receive transfers. Your 
+12-word recovery phrase will not change, so you don't need to back up your wallet again. Your rewards and transfer
+history will start from scratch.
+
+Otherwise, it would be best to switch to a new wallet manually. It means that you need to create a new wallet and transfer
+all your tokens there.
+Here is what you need to do:
+1. Make sure your current wallet is backed up! If not, backup your current wallet by going to Settings > Wallet Settings >
+Wallet backup
+2. Disconnect your current wallet by going to Settings > Disconnect Wallet
+4. Create a new wallet as you did it before
+5. Backup the new wallet
+6. Check your new wallet public key by clicking the "Receive" button and save it
+7. Switch to your current wallet by disconnecting the new one and importing the current using a 12-word recovery phrase
+8. Transfer your balance to the new wallet using your new public key by clicking the "Send" button
+9. Switch back to your new wallet again
+10. You are done! Continue enjoying the Nodle Cash app!
+
+### What if I received a transfer to a legacy wallet after being upgraded?
+
+No worries, your tokens are not lost! You can [re-import your legacy wallet to a web wallet](./reimport-your-account) and 
+transfer your tokens to your new wallet. For the import, you should use your current 12-word recovery phrase, but select 
+"Edwards" in the Advanced creation options selector.
```

Comment:
does this mean it will not be importable with the Nodle Cash app anymore?

Reply:
Yes, as we discussed before if the balance on the SR key > 0, we will proceed with it. So it will not be possible to import the ED key with the same mnemonics in the mobile app.

Reply:
Got it, I wasn't grasping the nuance ✅